### DISCUSSION TITLE
Possible to set which attribute to show in Features list in browse mode?

### DISCUSSION THREAD

**Discussion by 9ls1** [opengisch/QField#4231] [Q&A]
When selecting a feature in Browse mode,  one (or more) feature(s) are listed from one (or more) layer(s).
It seems like the attribute value of the first editable attribute (field) of widget type Text is default?
Is it possible to change this - to decide which attribute to display?

By default(?) the attribute value of the attribute "skydekke" (cloud coverage in %) is shown:
![Screenshot_20230425_141123](https://user-images.githubusercontent.com/10632163/234272596-a9d51c84-135a-4a4f-93e5-bb7476d6e154.jpg)

Is it possible to show another attribute in stead, e.g. ftid or use/add the attribute name as prefix? Just the attribute value may not always be too informative (e.g. skydekke 10 or ftid 3000-3)?
![Screenshot_20230425_141150](https://user-images.githubusercontent.com/10632163/234272612-c45057f7-0551-4e7c-b039-80ccc6b39428.jpg)

**Comment by nirvn** [comment]
You can customize that by changing the display name value in QGIS, see documentation: https://docs.qgis.org/3.28/en/docs/user_manual/working_with_vector/vector_properties.html#display-properties

**Reply by 9ls1** [reply]
@nirvn Thanks, works like a charm!
